src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,40 @@
+Manifest-Version: 1.0
```

Comment:
Do we need this file for the PR? Just asking because I'm not sure what the team would like for this file if we include one.